pallets/task/src/tests.rs

```diff
@@ -519,18 +518,6 @@ fn task_can_be_rejected_by_creator(){
 	});
 }
 
-#[test]
-fn transfer_balance_works(){
```

Comment:
Please add new tests!

Reply:
That test just test same thing which has been well tested in pallet_balances, or pallet_assets or orml_token etc.
So I don't thing we need that.